server/fixtures/secret/testDataA

```diff
@@ -0,0 +1 @@
+secretDataA
```

Comment:
Add return

Reply:
I don't think we need returns for secret data. (Typically I haven't seen it in secrets.)

Reply:
I agree with you @mrunalp . If we add the `return`,  this test will be more complicated. As we know,  return is represented `\n` in MacOS/Linux or `\r\n` in Windows. @rhatdan

Reply:
Ok, I did not know.  Live and learn.